main.cpp

```diff
@@ -184,20 +187,39 @@ class Grid {
     }
 };
 
+int main() {
+    std::ifstream inputFile("testbench/simple-50.txt"); // open file for input

```

Comment:
Can you make sure that the files are included in the PR? I can see the files on the branch, but not in the PR. Not sure if its just a folder thing